admin-site/src/Occasion.svelte

```diff
@@ -101,7 +101,6 @@ onMount(async () => {
         headers: {'Content-Type': 'application/json'},
         body: JSON.stringify({"state":"closed"}) 
       }).then( response => { 
-        console.log(response.status);
         if(response.status == 200){
           response.json().then( details => {
             //update state of occasion
```

Comment:
Usually I would use the data sent back in `details` to drive whatever happens next (i.e. setting occasionOpen to some value)

Reply:
Ah ok. So for example add conditions in there like: if(details.state == "opened"){ console.log("we are open") }?

Reply:
No, more that the next line `occasionOpen.set(false)` feels arbitrary. Like, it would be cleaner to have individual functions handle the open button / close button, and then a single function that calls the back end to update, and uses the returned object (and its fresh 'state' property) to drive what happens in response.

Reply:
ok, I think I know what you mean. I'll have a stab at it, and then shoot you a note for review.